keyboards/splitography/keymaps/default/keymap.c

```diff
@@ -0,0 +1,330 @@
+        QWERTY1, STN_S1,  STN_TL,  STN_PL,  STN_HL,  STN_ST1, STN_ST3, STN_FR,  STN_PR,  STN_LR,  STN_TR,  STN_DR ,
+        QWERTY2, STN_S2,  STN_KL,  STN_WL,  STN_RL,  STN_ST2, STN_ST4, STN_RR,  STN_BR,  STN_GR,  STN_SR,  STN_ZR ,
+                                            STN_A,   STN_O,   STN_E,   STN_U
+    ),
+};
+// clang-format on
+
+// ..................................................................... Keymaps
+
+#define QWERTY_1 1
+#define QWERTY_2 2
+#define QWERTY_12 3
+static uint8_t qwerty_n = 0;
+
+void qwerty(void) {
+    qwerty_n = 0;
+    layer_move(0);
+    set_single_persistent_default_layer(_QWERTY);
+}
+
+void plover(keyrecord_t *record) {
+    if (record->event.pressed) {
+        layer_move(0);
+        layer_on(_PLOVER);
+        if (!eeconfig_is_enabled()) {
+            eeconfig_init();
+        }
+        keymap_config.raw  = eeconfig_read_keymap();
+        keymap_config.nkro = 1;
+        eeconfig_update_keymap(keymap_config.raw);
+    }
+}
+
+void gemini(void) {
+    layer_move(0);
+    layer_on(_GEMINI);
+}
+
+// ........................................................... User Keycode Trap
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+        case QWERTY:
+            if (record->event.pressed) {
+                qwerty();
+            }
+            return false;
+        case QWERTY1:
+            if (record->event.pressed) {
+                qwerty_n = qwerty_n | QWERTY_1;
+                if (qwerty_n == QWERTY_12) {
+                    qwerty();
+                }
+            } else {
+                qwerty_n = qwerty_n & ~QWERTY_1;
+            }
+            return false;
+        case QWERTY2:
+            if (record->event.pressed) {
+                qwerty_n = qwerty_n | QWERTY_2;
+                if (qwerty_n == QWERTY_12) {
+                    qwerty();
+                }
+            } else {
+                qwerty_n = qwerty_n & ~QWERTY_2;
+            }
+            return false;
+        case BLUE:
+            if (record->event.pressed) {
+                layer_on(_BLUE);
+                update_tri_layer(_BLUE, _ORANGE, _GREEN);
+            } else {
+                layer_off(_BLUE);
+                update_tri_layer(_BLUE, _ORANGE, _GREEN);
+            }
+            return false;
+        case ORANGE:
+            if (record->event.pressed) {
+                layer_on(_ORANGE);
+                update_tri_layer(_BLUE, _ORANGE, _GREEN);
+            } else {
+                layer_off(_ORANGE);
+                update_tri_layer(_BLUE, _ORANGE, _GREEN);
+            }
+            return false;
+        case PLOVER:
+            plover(record);
+            return false;
+        case GEMINI:
+            if (record->event.pressed) {
+                gemini();
+            }
+            return false;
+    }
+    return true;
+}
+
+// initialize steno protocol
+void matrix_init_user(void) {
```

Comment:
```suggestion
void eeconfig_init_user(void) {
```

Reply:
This seems appropriate since the steno mode is stored in the eeprom, but the [documentation](https://beta.docs.qmk.fm/using-qmk/hardware-features/feature_stenography) puts the call to `steno_set_mode` inside `matrix_init_user`.

Reply:
The documentation is a bit out of date.  Also, this value does run on init, so it may end up writing to eeprom often.    `eeconfig_init_user` runs on initial flash, or whenever the eeprom is reset.  So it's definitely the better option here.